Task_2/src/Runner.java

```diff
@@ -0,0 +1,22 @@
+import model.Counter;
+import model.Text;
+
+
+import java.io.IOException;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class Runner {
+
+    public static void main(String[] args) throws IOException {
+        String text1 = new String(Files.readAllBytes(Paths.get("src\\text.txt"))).replace("\r", " ");

```

Comment:
Several responsibilities in one line, but it seems to be extracted to a method or class.
 `text1` what does it mean? Why 1?